locations/spiders/sears.py

```diff
@@ -88,15 +88,38 @@ def parse_sears_outlet_store(self, response):
         # Ensure ref is consistent across sears sites - 7 digit unique store identifier (zero padded)
         ref = str(store_data['unit']).zfill(7)
 
+        name = None
+        addr_full = None
+        city = None
+        state = None
+        postcode = None
+        phone = None
+        website = None
+
+        if 'storeName' in store_data:
```

Comment:
Suggestion: Instead of all these defaults and if statements, you can use the `get` method on the store_data dictionary. E.g.:

```
properties = {
    'ref': ref,
    'name': store_data.get('storeName'),
    'addr_full': store_data.get('streetAddr'),
    ...
```

The get method returns None if the property doesn't exist (unless you specify a different default value in the call)

Reply:
Done, Thanks!